modules/graceful-shutdown.adoc

```diff
@@ -42,7 +42,7 @@ $ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-s
 +
 [source,terminal]
 ----
-$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1 <1>; done
+$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1; done <1>
 ----
 <1> Indicates how long, in minutes, this process lasts before the control-plane nodes are shut down. For large-scale clusters with 10 nodes or more, set to 10 minutes or longer to make sure all the compute nodes have time to shut down first.
```

Comment:
I just reviewed this line in the later releases and I recall that "Indicates how long" was updated to "`-h 1` indicates how long" when the call out numbering was moved for line 45. You could update this line in this PR for 4.6 too.

[Here](https://github.com/openshift/openshift-docs/blame/main/modules/graceful-shutdown.adoc#L47) is an example of the updated call out in another branch, in this case main.

Reply:
Addressed